core/src/main/java/hudson/model/Computer.java

```diff
@@ -1774,6 +1774,14 @@ public long getWhen() {
                     Permission.WRITE,
                     PermissionScope.COMPUTER);
 
+    public static final Permission AGENT_SECRET =
```

Comment:
Do we really need a new permission? Isn't one of the existing permissions agent.connect or agent.configure sufficient?